lib/cylc/task_state.py

```diff
@@ -86,21 +86,33 @@
     TASK_STATUS_FAILED
 ])
 
-# Task statuses that are killable.
-TASK_STATUSES_KILLABLE = set([
-    TASK_STATUS_SUBMITTED,
-    TASK_STATUS_RUNNING
+# Task statuses that are final.
+TASK_STATUSES_FINAL = set([
+    TASK_STATUS_EXPIRED,
+    TASK_STATUS_SUCCEEDED,
+    TASK_STATUS_FAILED,
+    TASK_STATUS_SUBMIT_FAILED,
 ])
 
-# Task statuses that are externally active.
-TASK_STATUSES_ACTIVE = TASK_STATUSES_KILLABLE
+# Task statuses that are externally active, i.e. pollable and killable
+TASK_STATUSES_TO_BE_ACTIVE = set([
+    TASK_STATUS_QUEUED,
+    TASK_STATUS_READY,
+    TASK_STATUS_SUBMIT_RETRYING,
+    TASK_STATUS_RETRYING,
+])
```

Comment:
I guess the comment for this set needs to be altered a bit: `Tasks statuses that are TO BE externally active...`